content/applications/sales/sales/ebay_connector/manage.rst

```diff
@@ -1,50 +1,101 @@
+product form. These fields can be used to customize the configuration of the product, and how it
+appears on eBay.
 
-Listing with variations
-=======================
+.. image:: manage/sell-on-ebay-configurations.png
+   :align: center
+   :alt: The sell on ebay configurations under the ebay tab on a product form in Odoo sales.
 
-When the **use eBay** on a product with variations is checked and with **Fixed
-Price** as **Listing Type**, the eBay form is slightly different. In the
-variants array, you can choose which variant will be listed on eBay as well as
-set the price and the quantity for each variant.
+.. tip::
+   The :guilabel:`Description Template` allows the use of templates for eBay listings. To create a
+   template, type the desired name of the template in the :guilabel:`Description Template` field,
+   and click :guilabel:`Create and edit...` from the drop-down menu that appears.
 
-.. image:: manage/manage02.png
-  :align: center
+   When that's selected, a :guilabel:`Create Description Template` pop-up form appears, in which
+   the body of the description can be added, along with any attachments that may be necessary.
 
-Listing with item specifics
-===========================
+   .. image:: manage/create-description-template-popup-form.png
+      :align: center
+      :alt: The crete description template pop-up form in ebay tab on product form in Odoo sales.
 
-In order to add item specifics, you should create a product attribute with one
-value in the **Variants** tab on the product form.
+Listing (with product variants)
+===============================
 
-.. image:: manage/manage03.png
-  :align: center
+When the :guilabel:`Sell on eBay` feature is activated under the :guilabel:`eBay` tab on a product
+form for a product with variants, the available field configurations are the same as products
+without variants.
 
-Product Identifiers
+However, when the :guilabel:`Listing Type` us changed to :guilabel:`Fixed Price`, Odoo presents a
+variant table at the bottom of the :guilabel:`eBay` tab, in which the :guilabel:`Fixed Price` can be
+entered, the decision to :guilabel:`Publish on eBay` can be made for specific variants, along with
+other options.
+
+.. image:: manage/fixed-listing-price.png
+   :align: center
+   :alt: The fixed price listing type in the ebay tab on a product form in Odoo sales.
+
+Listing (with item specifics)
+=============================
+
+In order to add product specifics (e.g. brand names, etc.), a product attribute dedicated to that
+specific detail should be created -- with that detail represented by a single value -- in the
+:guilabel:`Attributes & Variants` tab of the product form.
+
+To do that, open the desired product form, and click into the :guilabel:`Attributes & Variants` tab.
+Then, click :guilabel:`Add a line`, name the :guilabel:`Attribute` after the product detail (e.g.
+`Brand`), and enter the specific detail in the :guilabel:`Values` column (e.g. `Apple`).
```

Comment:
Does this mean you can only add these specifics when selling at a fixed price? or does the attribute carry over with the auction price as well?

Reply:
Great question! It's my understanding that it can be added with/without a fixed price. And, since I don't exclusively say that it's for one or another, I think it'll still read that way for the user -- since they won't run into any conflicts when they attempt to do this.